### ISSUE TITLE
some glitches on the wall

### ISSUE THREAD

**Issue by johndoe71rus** [diasurgical/devilutionX#1271]
linux x64 (Calculate Linux and Lubuntu 20.04)

some glitches on the wall.
attach save and video.
test on 2 different pc and devilutionX version.

[vokoscreenNG-2021-03-25_08-36-57.zip](https://github.com/diasurgical/devilutionX/files/6202366/vokoscreenNG-2021-03-25_08-36-57.zip)

[art_single_0.zip](https://github.com/diasurgical/devilutionX/files/6202369/art_single_0.zip)

**Comment by AJenbo** [comment]
unable to play the video format, can you put some words to this

**Comment by johndoe71rus** [comment]
load save and move around. Look at area on the screenshots/
this is vp8 codec. next time try record n264
![Снимок экрана_2021-03-25_10-03-17](https://user-images.githubusercontent.com/7861172/112432434-f360b200-8d51-11eb-819d-b838a71de82f.png)
![Снимок экрана_2021-03-25_10-03-04](https://user-images.githubusercontent.com/7861172/112432445-f65ba280-8d51-11eb-8c42-5e247d5d7758.png)

**Comment by julealgon** [comment]
Isn't this the exact same issue as with caves? Also, doesn't it happen in vanilla as well?

**Comment by AJenbo** [comment]
Yes, Caves have similar issues, but it's not sure that it's the exact same tile that it's happening for. I put "Insufficient information" on it until we can verify it with vanilla, but I expect it to be there also.

**Comment by Chance4us** [comment]
The attached savegame can be loaded in vanilla and it is looking like following.
![screen00](https://user-images.githubusercontent.com/45902449/112513251-6db22600-8d94-11eb-916e-e65854edab6b.png)

**Comment by AJenbo** [comment]
Related https://github.com/diasurgical/devilutionX/issues/111
